<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorDialog</class>
 <widget class="QDialog" name="MotorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QTabWidget" name="motorTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Start">
       <attribute name="title">
        <string>Start</string>
       </attribute>
       <widget class="QProgressBar" name="motorProgressBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>291</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QPushButton" name="restore">
        <property name="text">
         <string>Restore Default</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MotorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MotorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
